sympy/solvers/bivariate.py

```diff
@@ -210,7 +210,7 @@ def _solve_lambert(f, symbol, gens):
         rhs = log(rhs)
 
     lhs = factor(lhs, deep=True)
-    # make sure we are inverted as completely as possible
+    # make sure we have inverted as completely as possible
```

Comment:
> Can it be:
make sure we have inverted the original equation f as completely as possible.

That doesn't look correct.

> or
> after some modification to the original equation f make sure we have inverted it as completely as possible

Hmm, I just looked through the code. Current comment looks fine.